courses/mushroom_cup/moo_moo_farm/course_data.inc.c

```diff
@@ -8691,7 +8691,7 @@ Gfx d_course_moo_moo_farm_dl_13F20[] =
     gsDPSetTile(G_IM_FMT_CI, G_IM_SIZ_8b, 4, 0x0000, G_TX_RENDERTILE, 0, G_TX_NOMIRROR | G_TX_CLAMP, 6, G_TX_NOLOD, G_TX_NOMIRROR | G_TX_CLAMP, 5, G_TX_NOLOD),
     gsDPSetTileSize(G_TX_RENDERTILE, 0, 0, 0x007C, 0x00FC),
     gsDPLoadTextureBlock(0x03009000, G_IM_FMT_CI, G_IM_SIZ_8b, 32, 64, 0, G_TX_NOMIRROR | G_TX_CLAMP, G_TX_NOMIRROR | G_TX_CLAMP, G_TX_NOMASK, G_TX_NOMASK, G_TX_NOLOD, G_TX_NOLOD),
-    gsSPVertex(d_course_moo_moo_farm_mole_model, 8, 0),
+    gsSPVertex(d_course_moo_moo_farm_tee_model, 8, 0),
```

Comment:
tee? Is that the name of the mole? Or was this unrelated to mole?

Reply:
It should be tree